validation/src/test/scala/hmda/validation/rules/lar/macro/Q016Spec.scala

```diff
@@ -0,0 +1,45 @@
+package hmda.validation.rules.lar.`macro`
+
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+
+class Q016Spec extends MacroSpec {
+
+  val config = ConfigFactory.load()
+  val multiplier = config.getDouble("hmda.validation.macro.Q016.numOfLarsMultiplier")
+  val incomeCap = config.getInt("hmda.validation.macro.Q016.incomeCap")
+
+  val testLars = lar100ListGen.sample.getOrElse(Nil)
+  val sampleSize = testLars.size
+  val sampleSizeTarget = (testLars.size * multiplier).toInt
+  def underCap(lar: LoanApplicationRegister) = {
+    val underCapLoan = lar.loan.copy(amount = 9)
```

Comment:
I think it would be good to move this and the `overCapLoan` to a function of the `incomeCap` defined earlier.